setup.py

```diff
@@ -18,7 +18,7 @@
     author="Yoshiki Vazquez-Baeza",
     author_email="yoshiki@ucsd.edu",
     description="Display ordination plots",
-    license="bsd",
+    license='BSD-3-Clause',
```

Comment:
I think the way to write this (from our interactions with the conda-forge team) is `BSD Clause-3`.